stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,88 @@
+
+[Hangman](https://en.wikipedia.org/wiki/Hangman_(game)) is a classic word game in which you must find the correct answer by guessing guessing letters one at a time.
+
+### Game rules
+1. The game is represented by two main parts: the gallows part and the quiz part.
+2. At the beginning of the game, the gallows is empty.
+3. The quiz part includes:
+- a question,
+- a secret word (the answer to the question) represented by the blank lines for each letter in the word *(for example, if the secret word is 'address' it will be written like '_ _ _ _ _ _ _')*,
+- an incorrect guesses counter,
+- a virtual keyboard,
+4. The user should be able to use **both virtual** *(by clicking letter buttons)* and **physical keyboards** *(by pressing letter keys)* for playing the game.
+5. When the user starts guessing the letters:
+- whenever the letter is guessed correctly, this letter appears instead of the corresponding blank line *(for the example mentioned above, if the user clicks or press on the 'd' letter, both 'd's should appear on the screen: '_ d d _ _ _ _')*;
+- if the letter is not in the word, **one body part** should be added to the gallows and the incorrect guesses counter should be updated accordingly;
+- the clicked/pressed letter should be disabled and it should be reflected in the UI *(for example, by using a different color on the disabled button)*;
+6. The body parts appear on the gallows in the following order: head, body, left arm, right arm, left leg, right leg. It means that the user has *6 attempts* to guess the word.
+7. The player will continue guessing letters until either the word is solved or all six body parts are on the gallows.
+8. At the end of the game, a modal window should appear. It should:
+- include the message that either congratulates or informs the user that the game has been lost;
+- the secret word (independently whether it was guessed correctly or not);
+- a "play again" button.
+9. When the user clicks on "play again", the game should start over:
+- the gallows should be empty;
+- the question should be changed;
+- the blank lines should be updated according to the new secret word.
+- an incorrect guesses counter should be set to 0.
+
+### Approximate appearance
+*This example is purely illustrative. Feel free to change the layout as you wish.*
+![screenshot](hangman.png)
+
+### Helpful materials
+At the [following link](https://www.figma.com/file/ug2NAUiXPpaFDvch5TWUxd/Hangman-game?type=design&node-id=0%3A1&mode=design&t=4Gj7Ayo0yckppNo8-1) you will find:
+- examples of the game flow,
+- body parts (for those who don't want to design them by themselves).
+
+### Main functional requirements
+- The design should be adaptive (or responsive), that includes desktop(1440px <= width), tablet(768px <= width < 1440px) and mobile(360px <= width < 768px). When switching between versions everything should be displayed correctly, all functionality should be present, nothing should disappear or leave beyond the screen.
```

Comment:
_The same suggestion to use the word "must" instead of "should"_
_Also a small grammar correction with "left beyond the screen"_

**Suggested:** - The design must be adaptive (or responsive), that includes desktop(1440px <= width), tablet(768px <= width < 1440px) and mobile(360px <= width < 768px). When switching between versions everything must be displayed correctly, all functionality must be present, no content must disappear or be left beyond the screen.

Reply:
Done